### PR TITLE
doc: Correct PVC mention in the deployment doc

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@nixpanic ptal.

File: docs/deploy-rbd.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ be able to review jwt tokens.
 Configure a role(s) for service accounts used for ceph-csi:
 
 * provisioner service account (`rbd-csi-provisioner`) requires only **delete**
-  permissions to delete passphrases on pvc delete
+  permissions to delete passphrases on PVC delete
```
Comment (Madhu-1) [inline docs/deploy-rbd.md:255]: can you also update expand-pvc.md and snapshot.md

Comment (humblec) [reply docs/deploy-rbd.md:255]: Oh.. sure.

**Comment by humblec** [comment]
@nixpanic can you check this trivial PR ?

**Comment by humblec** [comment]
@Madhu-1 can we add `ready-to-merge` here?

**Pull request merged**